api/applications/applicationModel.js

```diff
@@ -1 +1,45 @@
-// TODO: Refactor these function to fetch data from tickets table instead of application_tickets table
+const db = require('../../data/db-config');
+
+//this is a basic model for our application flow. Currently, imports from applicationMiddleware depend on it and should be working correctly now.
```

Comment:
Hey why do we still need this comment?

Reply:
I thought to be helpful going forward, but looking at it now you are right it is unneeded. Will excise it shortly!